tests/build-and-run-tests.sh

```diff
@@ -28,14 +28,23 @@ function ciSkip {
     exit 0
 }
 
+# Finds the branching point of two commits. 
+# For example, let B and D be two commits, and their ancestry graph as A -> B, A -> C -> D. 
+# Given commits B and D, it returns A. 
+function getLowestCommonAncestorCommitHash {
```

Comment:
`getBranchingPoint` is better or even `getAncestorCommit`, I was looking for something shorter.